Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  OptionsView.swift
+//  Rear Rider
+//
+//  Created by Bobby Palko on 10/25/22.
+//
+
+import SwiftUI
+
+// This binding extension lets us attach our change handler directly to the binding instead of the view, letting us place the observer next to the thing its observing instead of having onChange() modifiers everywhere.
+extension Binding {
+    func onChange(_ handler: @escaping (Value) -> Void) -> Binding<Value> {
+        Binding(
+            get: { self.wrappedValue },
+            set: { newValue in
+                self.wrappedValue = newValue
+                handler(newValue)
+            }
+        )
+    }
+}
+
+struct OptionsView: View {
+    @ObservedObject var conf = UserConfig()
+    @State var confAudio: String = ""
+    @State var confLightPattern: String = ""
+    @State var confLightBrightness: Int = 1
+    @State var confLightColor: Color = .white
+    
+    let audioFiles: [ConfigOptions.AudioFile] = ConfigOptions.AudioFile.allCases
+    let lightPatterns: [ConfigOptions.LightPattern] = ConfigOptions.LightPattern.allCases
+    let lightBrightness: [ConfigOptions.LightBrightness] = ConfigOptions.LightBrightness.allCases
+    
+    let alert = RearRiderAlerts()
```

Comment:
This one as well should be a StateObject in Rear_RiderApp.

Reply:
Moving the Config into a StateObject makes sense, but I disagree on the Alerts here. These are only being initialized when we play a sound; we don't need this object active at all times. Especially since the config is handling the filename and everything - this just plays it